robottelo/cleanup.py

```diff
@@ -85,7 +85,9 @@ def register_entity_for_cleanup(self, sender, entity, **kwargs):
 
     def clean(self):
         """This method is called in TearDownClass only when cleanup=true"""
-        default_org = entities.Organization(id=DEFAULT_ORG_ID)
+        default_org_id = entities.Organization().search(
+            query={'search': 'name="{}"'.format(DEFAULT_ORG)})[0].id
```

Comment:
I thought if we used this line across multiple times, we can create a common function. e.g. 

default_org or default_org_id 

```python 
def get_default_org:
        return entities.Organization().search(query={'search': 'name="{}"'.format(DEFAULT_ORG)})[0].id
```
what do you think ?

Reply:
your point is valid .. I checked on that but somehow did not find any direct way where it could use directly call by all place where this dependency requires to remove hence wannted to go with same.